charts/newrelic-prometheus-agent/Chart.yaml

```diff
@@ -4,13 +4,13 @@ description: A Helm chart to deploy Prometheus with New Relic Prometheus Configu
 
 type: application
 
-version: 0.0.4
+version: 0.0.5
 
 # Prometheus Server Version
 appVersion: "v2.37.1"
 annotations:
 # TODO replace with the first configurator release
-  configuratorVersion: "0.0.1"
+  configuratorVersion: "0.0.2"
```

Comment:
Is there a task to automate this using renovatebot?

I am rethinking if we should swap `appVersion` and this `configurationVersion` but we still do not know to automate the upgrade of Prometheus Agent. Nevertheless, this is a NIT.

Reply:
> I am rethinking if we should swap appVersion and this configurationVersion but we still do not know to automate the upgrade of Prometheus Agent. Nevertheless, this is a NIT.

We have discussed about this and decided that from the user expectation point of view the appVersion should be the Prometheus one. I guess the discussion is around some PR or slack